.gitignore

```diff
@@ -2,5 +2,6 @@ RAMCloud
 RAMCloud-install
 .vscode
 tmp
+tmp_*
```

Comment:
Why are `tmp` and `tmp_*` in the gitignore? Does some part of the repository create these? If these are things that you personally want ignored but they aren't created by some normal procedure of using the repository, you can have a custom global gitignore to ignore certain files only for you. (See https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore). Technically `.vscode` is also something that probably shouldn't be committed, since vscode isn't part of the project, but enough people use vscode that it's a nice thing to have.

Reply:
Making local .git/info/exclude for tmp, tmp_* and .vscode

The tmp directories are made by me and not anyone else